plugins/BEdita/Core/src/Utility/ProjectModel.php

```diff
@@ -122,11 +123,23 @@ protected static function relations(): array
                 $right = (array)Hash::extract($row, 'right_object_types.{n}.name');
                 sort($left);
                 sort($right);
-                unset($row['id']);
-                $row->set('left_object_types', $left);
-                $row->set('right_object_types', $right);
+                $row->unset(['id', 'left_object_types', 'right_object_types']);
+                $row->setAccess(['left', 'right'], true);
+                $row->set(compact('left', 'right'));
```

Comment:
Maybe instead of changing property accessibility we could simply do
```php
$row->set('left', $left);
$row->set('right', $right);
```

Reply:
ok, the problem was the `guard` option set to `true` on a mass assignment using `compact()` 

done